nomenclature/testing.py

```diff
@@ -2,16 +2,34 @@
 import logging
 from pathlib import Path
 
+import click
+
+import nomenclature
 
 logger = logging.getLogger(__name__)
 
+cli = click.Group()
+
+
+@cli.command("validate-yaml")
+@click.argument("path", type=click.Path(exists=True, path_type=Path))
+def cli_valid_yaml(path: Path):
+    assert_valid_yaml(path)
+
+
+@cli.command("validate-project")
+@click.argument("path", type=click.Path(exists=True, path_type=Path))
+def cli_valid_structure(path: Path):
```

Comment:
Minor suggestion for clarity (otherwise, it's a bit confusing with `assert_valid_structure`...
```suggestion
def cli_valid_project(path: Path):
```